pylinac/winston_lutz.py

```diff
@@ -366,6 +366,10 @@ def __init__(self, file):
         self.check_inversion()
         # Flip the image upside down... makes it more intuitive later on.
         self.flipud()
+        # The first pixel line on the left has a glitch on Elekta iview EPID
```

Comment:
No problem. Once you confirm that and upload a dataset with the issue I'll merge.